rtf_vis_tool/src/Components/frustum_classes/view_frustum.js

```diff
@@ -0,0 +1,170 @@
+/* Contains functions to transform camera instrinsic variables into a frustum consisting of 5 vertices.
+
+Author: Adi Singh
+*/
+
+// Third-Party Package Imports.
+var nj = require('numjs');
+
+class ViewFrustum {
```

Comment:
thanks for the PR, @adisingh50 , this looks great.

Could you port the Python unit tests for this class into JS unit tests, as well? Here: https://github.com/borglab/gtsfm/blob/master/tests/common/test_view_frustum.py

And can we add to the CI script the commands to run the JS unit tests in CI?

Reply:
Just added a few more unit tests to `gtsfm/rtf_vis_tool/tests`. I have also updated the README within `rtf_vis_tool` to show the command which runs all the unit tests (step 4). The command is `npm test a`.

Reply:
thanks. Can we put that line into the CI file also, in `.github/scripts/blah.sh`?